### PR TITLE
FOGL-2206 Add display names and order

### PR DESCRIPTION
**Pull request by MarkRiddoch**

### CODE REVIEW

File: plugin.cpp [hunk 1]
```diff
@@ -20,14 +20,16 @@
...
                        		"\"type\" : \"string\", " \
-				"\"default\" : \"" FILTER_NAME "\" }, " \
+				"\"default\" : \"" FILTER_NAME "\", \"readonly\" : \"true\" }, " \
 			 "\"enable\": {\"description\": \"A switch that can be used to enable or disable execution of " \
 					 "the asset filter.\", " \
+				"\"displayName\": \"Enable\", " \
 				"\"type\": \"boolean\", " \
 				"\"default\": \"false\" }, " \
-			"\"config\" : {\"description\" : \"Asset C filter configuration.\", " \
+			"\"config\" : {\"description\" : \"JSON docuemnt that defines the rules for asset names.\", " \
```
Comment (AmandeepArora) [inline plugin.cpp]: Typo: It should be "document"

**Review by AmandeepArora — DISMISSED** [review]
Looks good. Just a small typo.

File: plugin.cpp (hunk 1 above)
Comment (MarkRiddoch) [reply plugin.cpp]: Thanks

File: plugin.cpp [hunk 2]
```diff
@@ -20,14 +20,16 @@
 
 #define DEFAULT_CONFIG "{\"plugin\" : { \"description\" : \"Asset filter plugin\", " \
                        		"\"type\" : \"string\", " \
-				"\"default\" : \"" FILTER_NAME "\" }, " \
+				"\"default\" : \"" FILTER_NAME "\", \"readonly\" : \"true\" }, " \
 			 "\"enable\": {\"description\": \"A switch that can be used to enable or disable execution of " \
 					 "the asset filter.\", " \
+				"\"displayName\": \"Enable\", " \
```
Comment (shariq1501) [inline plugin.cpp]: change displayName Enable to Enabled

**Comment by MarkRiddoch** [comment]
This pull request is showing incorrect changes. Closing and recreating.

**Pull request closed without merging**